components/admin_console/license_settings/modals/upload_license_modal.tsx

```diff
@@ -220,7 +222,8 @@ const UploadLicenseModal = (props: Props): JSX.Element | null => {
             </div>
         </>
     );
-    if (currentLicense.IsLicensed === 'true') {
+
+    if (uploadSuccessful) {
```

Comment:
Question, if the upload fails, does it still have the previous license still available?

Reply:
@marianunez yes it is still available.